celesta-java/src/test/java/ru/curs/celesta/bad/annotated/twocallcontexts/TwoCallContexts.java

```diff
@@ -0,0 +1,10 @@
+package ru.curs.celesta.bad.annotated.twocallcontexts;
+
+import ru.curs.celesta.CallContext;
+import ru.curs.celesta.annotation.CelestaProc;
+
+public class TwoCallContexts {
+
+    @CelestaProc
+    public void twoCallContexts(CallContext callContext1, CallContext callContext2) {};
```

Comment:
Same comment as above, additionally we could assertSame(callContext1, callContext2)

Reply:
We allow to declare only one CallContext argument in CelestaProc and it must be the first in order.
This CelestaProc is intended to check, that CelestaProcProvider throws CelestaException when declaration rule is not met.